marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/impl/security/getAuthoritiesTest.sjs

```diff
@@ -62,4 +49,13 @@ hubTest.runWithRolesAndPrivileges(['hub-central-mapping-writer'], [], function()
     assertions.push(test.assertTrue(authorities.includes('readMapping'), 'hub-central-mapping-writer should have "readMapping"'));
     assertions.push(test.assertTrue(authorities.includes('writeMapping'), 'hub-central-mapping-writer should not have "writeMapping"'));
 });
+
+// Test hub-central-saved-query-writer
```

Comment:
I don't think we want these assertions here. Instead, what we care about is that the EntitySearchController endpoints are secured correctly. Those should have @Secure("ROLE_manageSavedQuery") on them. You can then more effectively test the logic below by trying to hit the endpoints in a subclass of AbstractMvcTest as a user with that the saved-query roles, and without them.